packages/freesewing.shared/components/workbench/menu/core-settings/core-setting-only.js

```diff
@@ -21,7 +23,7 @@ const CoreSettingOnly = props => {
   return (
     <div className="py-4 mx-6 border-l-2 pl-2">
       <p className="m-0 p-0 px-2 mb-2 text-neutral-content opacity-60 italic">
-        {props.app.t(`settings.only.description`)}
+        {t(`settings:only.d`)}
```

Comment:
These changes are because I migrated all translation from react-intl to next-i18next, which uses a different way (better IMHO) to reference the keys.

The `settings:` part (called a namespace in i18next) is optional when:

 - It is either the default namespace (which is `app` in freesewing)
 - it is the only namespace imported by `useTranslation` in the component
 
 Since here we import three namespaces (`app`, `parts`, and `settings`) it is mandatory.

Reply:
It was just about the backticks which try to evaluate a string template, 
and here we have no variables in it.

Reply:
You are right of course, seems I commented on the wrong line here